src/person/Gender.java

```diff
@@ -0,0 +1,19 @@
+package person;
+
+/**
+ * 
+ * 
```

Comment:
Remove unused spaces and documentation to this enum class

Reply:
The issue that you have raised has been resolved